dist/js/apps/events-page/featuredEventsWidgetApp.js

```diff
@@ -152,13 +152,14 @@
 		};
 
 		var setStartDateForOnGoingEvent = function setStartDateForOnGoingEvent(signupEvent, currentDate) {
-			var cleanOnGoingStartDate = signupEvent.OnGoingStartDate.replace('T', ' ');
+			var cleanOnGoingStartDate = moment(signupEvent.OnGoingStartDate.replace('T', ' '));
```

Comment:
I really meant to ask this question here, why are you removing T?

Reply:
It's gone.